File: HumanResource/Models/Company.cs
using System.ComponentModel.DataAnnotations;

namespace HumanResource.Models
{
    public class Company
    {
        [Key] 
        public int Id { get; set; }
        public string CompanyName { get; set; }
        public string AddressLine1 { get; set; }
        public string? AddressLine2 { get; set; }
        public string City { get; set; }
        public string PostalCode { get; set; }
        public string CompanyEmail { get; set; }
        public string CompanyContact { get; set; }
        public string Website { get; set; }
        public string? AdminIdentifire { get; set; }

        #region Audit Field

        public bool IsDeleted { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }

        #endregion
    }

}

File: ClientApp/src/app/models/Company.ts
export default interface Company{
    id?: number;
    companyName?: string;
    addressLine1?: string;
    addressLine2?: string;
    city?: string;
    postalCode?: string;
    companyEmail?: string;
    companyContact?: string;
    website?: string;

    adminIdentifire?: string | any;

    isDeleted?: boolean | any;
    createdOn?: any;
    updatedOn?: any;
}